content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -209,11 +209,11 @@ well as lower-trust users.The following listed controls should be enforced/disal
 		<tr>
 			<td>Privilege Escalation</td>
 			<td>
-				Privilege escalation to root should not be allowed.<br>
+				Privilege escalation (such as via set-user-ID or set-group-ID file mode) should not be allowed.<br>
 				<br><b>Restricted Fields:</b><br>
-				spec.containers[*].securityContext.privileged<br>
-				spec.initContainers[*].securityContext.privileged<br>
-				<br><b>Allowed Values:</b> false, undefined/nil<br>
+				spec.containers[*].securityContext.allowPrivilegeEscalation<br>
```

Comment:
@tabbysable , From a docs perspective, the changes seem okay.

Reply:
I assume these fields and types are taken from:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core ?

Reply:
Exactly!

Reply:
/lgtm